### PR TITLE
Feat/vm prep

### PR DESCRIPTION
**Pull request by WeepingClown13**
- Updated the redundant privilege escalations in playbooks
- Improved logging
- Separated vm preparation and payload execution
- Change in certain paths
- Updated functions
- Prepare button added

### CODE REVIEW

File: ansible/aws/aws_env_setup_ubuntu_18.04.yaml [hunk 1]
```diff
@@ -1,74 +1,54 @@
...
-    become: yes
+      ./aws/install --update
+    args:
+      warn: false
+
   - block: 
     - name: Upgrade to AWS linux kernel
       apt:
-        name: linux-aws-lts-18.04
+        name: linux-image-generic-hwe-18.04
```
Comment (iamvishnuks) [inline ansible/aws/aws_env_setup_ubuntu_18.04.yaml:25]: did u tested this kernel?

Comment (WeepingClown13) [reply ansible/aws/aws_env_setup_ubuntu_18.04.yaml:25]: Yes

File: app.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 #from quart_jwt_extended import JWTManager
 
 class Settings(BaseSettings):
-    MONGO_DB: str = "mongodb://root:example@mongo:27017/"
+    MONGO_DB: str = "mongodb://root:example@localhost:27017/"
```
Comment (iamvishnuks) [inline app.py:20]: this should be mongo when we build container

File: routes/blueprint.py [hunk 3]
```diff
@@ -143,6 +145,19 @@ async def create_blueprint(data: BlueprintCreate, current_user: TokenData = Depe
     con.shutdown()
     return jsonable_encoder({"msg":"Succesfully updated","status":200})
 
+class Prepare(BaseModel):
+    project: Union[str,None] = None
+
```
Comment (iamvishnuks) [inline routes/blueprint.py:150]: we should accept hosts here

**Review by iamvishnuks — APPROVED** [review]

**Pull request merged**